third_party/docfx/templates/devsite/token.json

```diff
@@ -53,4 +53,4 @@
   "tip": "<h5>Tip</h5>",
   "important": "<h5>Important</h5>",
   "caution": "<h5>Caution</h5>"

```

Comment:
Do we also need to update these? I'm not sure where they're used.

Reply:
done!